docs/adr/20220308-namespaced-installation.md

```diff
@@ -0,0 +1,25 @@
+# Namespaced installation
+
+Date: 2022-03-08
+
+## Status
+
+Accepted
+
+Supersedes [Release name suffix](20211025-release-name-suffix.md)
+
+## Context
+
+Build tasks are not easily customizable (e.g. with respect to resource consumption/quotas or sidecars) in a centralized deployment model. In addition, for every change to a task, cluster admin rights are required.
+
+## Decision
+
+Abolish a central installation altogether. Provide images from a central registry (ghcr.io).
+
+## Consequences
+
+Every project needs to build images themselves, resulting in the installation being 
```

Comment:
maybe we should note that since we can use pre-built images from GitHub, this is not a huge issue